src/Compilers/CSharp/Portable/Lowering/Instrumentation/DynamicAnalysisInjector.cs

```diff
@@ -129,6 +134,79 @@ private static bool IsExcludedFromCodeCoverage(MethodSymbol method)
             return false;
         }
 
+        private static BoundExpressionStatement GetCreatePayloadStatement(
+            ImmutableArray<SourceSpan> dynamicAnalysisSpans,
+            BoundStatement methodBody,
```

Comment:
>BoundStatement methodBody [](start = 12, length = 25)

It looks like this is used only to get syntax, consider passing the syntax instead.